core/modules/messages/server/messages.js

```diff
@@ -96,6 +96,22 @@ Meteor.publish('messages', function (channel) {
   return Messages.find({ channel }, { sort: { createdAt: -1 }, limit });
 });
 
+Meteor.publish('allMessages', () => {
```

Comment:
nitpicking: I'd rather rename this record set "messages" and the previous one "channelMessages"